### DISCUSSION TITLE
有没有怎么做视频源的教程

### DISCUSSION THREAD

**Discussion by liuxingyin** [xiaohucode/yidaRule#219] [General]
有没有怎么做视频源的教程

**Comment by chenxingmoonset** [comment]
和文本源类似，最终获取的正文是视频直链就可以了

**Comment by liuxingyin** [comment]
大佬能有教程吗？有的话发一些，学习一下！
&nbsp;


流星银
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月26日(星期一) 下午4:37
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [xiaohucode/yidaRule] 有没有怎么做视频源的教程 (Discussion #219)















和文本源类似，最终获取的正文是视频直链就可以了
